src/WFS.js

```diff
@@ -163,14 +169,127 @@ L.WFS = L.FeatureGroup.extend({
 
         return that;
       },
-      error: function(errorMessage) {
+      error: function (errorMessage) {
         that.fire('error', {
           error: new Error(errorMessage)
         });
 
         return that;
       }
     });
+  },
+
+  getCapabilities: function (successCallback, errorCallback) {
```

Comment:
Unit tests for 'getCapabilities' are necessary. To save time you could get them from Leaflet-WMS plugin and adapt them to this plugin.